### PR TITLE
[docs] Encryption docs update

### PR DESCRIPTION
**Pull request by kaitlincarter-hc**
Bad link in encryption docs

### CODE REVIEW

**Comment by kaitlincarter-hc** [comment]
@freddygv I updated the certificate guide in the Learn repo to fix the "section" call out at the same time I created this PR: https://github.com/hashicorp/learn/pull/557/files. 

I just pushed an update to the encryption doc with your suggestions. 

There were several reasons we condensed the guides, mainly there was a lot of content overlap and it created the need to update the same information in two locations. Additionally, the difference between a new and existing was one rolling restart of the servers. I will investigate updating the call out boxes and possibly adding anchors so they're easier to link to.

**Comment by judithpatudith** [comment]
Just want to call out that #6071 might overlap with some of those issues. If so @kaitlincarter-hc you can go ahead and close that one in favor of this one 🙂

File: website/source/docs/agent/encryption.html.md [hunk 1]
```diff
@@ -101,7 +101,8 @@ and is secured using a symmetric key. See above for enabling gossip encryption.
 ## Configuring TLS on an existing cluster
 
 As of version 0.8.4, Consul supports migrating to TLS-encrypted traffic on a running cluster
-without downtime. This process assumes a starting point with no TLS settings configured, and involves
-an intermediate step in order to get to full TLS encryption. Review this step-by-step
-[guide](https://learn.hashicorp.com/consul/security-networking/agent-encryption#enable-tls-existing-cluster) to learn how. 
+without downtime. This process assumes a starting point with no TLS settings configured and involves
+an intermediate step in order to get to full TLS encryption. Review the
+[Securing RPC Communication with TLS Encryption guide](https://learn.hashicorp.com/consul/security-networking/certificates) to learn how. This guide details the step-by-step process for configuring TLS on a new or existing cluster. Note the call outs for specific information for
```
Comment (kaitlincarter-hc) [inline website/source/docs/agent/encryption.html.md]:
```suggestion
[Securing RPC Communication with TLS Encryption guide](https://learn.hashicorp.com/consul/security-networking/certificates) for the step-by-step process to configure TLS on a new or existing cluster. Note the call outs there for existing cluster configuration. 
```

File: website/source/docs/agent/encryption.html.md [hunk 2]
```diff
@@ -101,7 +101,8 @@ and is secured using a symmetric key. See above for enabling gossip encryption.
...
+an existing cluster.
```
Comment (kaitlincarter-hc) [inline website/source/docs/agent/encryption.html.md]:
```suggestion
```

**Pull request merged**